packages/docusaurus-theme-redoc/src/index.ts

```diff
@@ -14,14 +13,15 @@ export default function redocTheme(
 ): Plugin<null> {
   return {
     name: 'docusaurus-theme-redoc',
-    configureWebpack(_config, isServer) {
+    configureWebpack(_config, isServer,{currentBundler}) {
+      const bundler = currentBundler.instance ?? require("webpack")
```

Comment:
This makes the plugin compatible with Rspack, while keeping retrocompatibility with older versions